library/Rules/EndsWith.php

```diff
@@ -33,7 +62,7 @@ public function validate($input): bool
         return $this->validateEquals($input);
     }
 
-    protected function validateEquals($input)
+    protected function validateEquals($input): bool
```

Comment:
```diff
--    protected function validateEquals($input): bool
++    private function validateEquals($input): bool
```

Since they class is final.